lib/documentLoader.js

```diff
@@ -88,8 +87,8 @@ api.documentCache = async url => {
 
 api.loaders = [api.documentCache];
 
-function _createDidLoader({hostname, mode = 'test'}) {
-  didIo.use('v1', v1.driver({hostname, mode}));
+function _createDidLoader({mode = 'test'}) {
+  didIo.use(didVeresOne.driver({mode}));
```

Comment:
This `didIo` comes from `bedrock-did-io` which is already configured to use the veres-one driver and the mode for that driver is configured via the bedrock-did-io config.

See:
1. https://github.com/digitalbazaar/bedrock-did-io/blob/main/lib/config.js
2. https://github.com/digitalbazaar/bedrock-did-io/blob/main/lib/main.js

Reply:
Updated here f36950d